### PR TITLE
Condense tables

### PR DESCRIPTION
**Pull request by benja-matic**
This removes several tables from the results folder. I removed finetuning info from the milestones table, since the main use fo that table is to access the eval loss of pretrained models. The main results tables contain condensed results for bert base and small bert. Both are available in md and csv format. Condensed means for each finetuning task, the reported result is the best of all runs across a) finetuning prior to hp search and b) finetuning after hp search.

### CODE REVIEW

**Review by lucasosouza — APPROVED** [review]
Thanks!
